### PR TITLE
AG-1398 Fix for bug with pValues not showing when value is zero

### PR DESCRIPTION
**Pull request by sagely1**

### CODE REVIEW

File: src/app/features/genes/components/gene-comparison-tool/components/gene-comparison-tool-details-panel/gene-comparison-tool-details-panel.component.html [hunk 1]
```diff
@@ -15,11 +15,11 @@
     <div class="gct-details-panel-data">
       <div>
         <div>{{ data.valueLabel }}</div>
-        <div *ngIf="data.pValue">P-Value</div>
```
Comment (sagely1) [inline src/app/features/genes/components/gene-comparison-tool/components/gene-comparison-tool-details-panel/gene-comparison-tool-details-panel.component.html:18]: In cases when this variable was zero, the div and value below would not show.

File: src/app/features/genes/components/gene-comparison-tool/components/gene-comparison-tool-details-panel/gene-comparison-tool-details-panel.component.ts [hunk 2]
```diff
@@ -96,6 +96,9 @@ export class GeneComparisonToolDetailsPanelComponent {
   }
 
   getSignificantFigures(n: any, b: any) {
+    const emdash = '&#8212;'; // Shift+Option+Hyphen
```
Comment (sagely1) [inline src/app/features/genes/components/gene-comparison-tool/components/gene-comparison-tool-details-panel/gene-comparison-tool-details-panel.component.ts:99]: emdash is being used to visually expose missing data.

Comment (sagely1) [reply src/app/features/genes/components/gene-comparison-tool/components/gene-comparison-tool-details-panel/gene-comparison-tool-details-panel.component.ts:99]: A future update will move this into a custom pipe

**Review by JessterB — APPROVED** [review]
lgtm!

**Pull request merged**